src/hotspot/share/gc/shenandoah/heuristics/shenandoahPassiveOldHeuristics.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2018, 2019, 2021, Red Hat, Inc. All rights reserved.
```

Comment:
Same as above, only use first and last years.

Reply:
Fixed in next commit.